api/test/test_helper/fake_android_plugin.gd

```diff
@@ -127,3 +164,30 @@ func cancel(pack_names : Array):
 			return_asset_pack_states[PlayAssetPackStates._PACK_STATES_KEY][pack_name] = current_asset_pack_dict
 
 	return return_asset_pack_states
+
+# -----------------------------------------------------------------------------
+# Simulates the showCellularDataConfirmation() function in PlayAssetDelivery 
+# Android plugin. Emits signal with arguments configured using 
+# set_show_confirmation_response().
```

Comment:
Since we need to test exception cases that can not be easily triggered with normally, I used the preconfigured signal approach for these two functions.